content/github/writing-on-github/creating-gists.md

```diff
@@ -70,7 +70,7 @@ You can also create a gist using the {% data variables.product.prodname_cli %}.
 3. Type an optional description and name for your gist.
 ![Gist name description](/assets/images/help/gist/gist_name_description.png)
 
-4. Type the text of your gist into the gist text box.
+4. Type the text of your gist into the gist text box. Alternatively, you can drag and drop a text file from your desktop directly into the editor.
```

Comment:
Please move the sentence about drag and drop to the note above that contains the CLI information.

Reply:
I've gone ahead and made this change! The rest looks good so we will merge!